src/content/quickstart/verify_with_discovery.md

```diff
@@ -0,0 +1,26 @@
+####  Verify Deployment
+To verify the deployment follow the steps below.
+
+```bash
+export KUBECONFIG=cluster-b/auth/kubeconfig
+kubectl -n default create deployment nginx --image=nginxinc/nginx-unprivileged:stable-alpine
+kubectl -n default expose deployment nginx --port=8080
+kubectl -n default apply -f - <<EOF
+apiVersion: lighthouse.submariner.io/v2alpha1
+kind: ServiceExport
+metadata:
+  name: nginx-demo
```

Comment:
This should be nginx instead of nginx-demo?

Reply:
yes! thanks